pallets/asset/src/lib.rs

```diff
@@ -893,6 +901,42 @@ decl_module! {
         pub fn remove_ticker_pre_approval(origin, ticker: Ticker) -> DispatchResult {
             Self::base_remove_ticker_pre_approval(origin, ticker)
         }
+
+        /// Sets all identities in the `mediators` set as mandatory mediators for any instruction transfering `ticker`.
+        ///
+        /// # Arguments
+        /// * `origin`: The secondary key of the sender.
+        /// * `ticker`: The [`Ticker`] of the asset that will require the mediators.
+        /// * `mediators`: A set of [`IdentityId`] of all the mandatory mediators for the given ticker.
+        ///
+        /// # Permissions
+        /// * Asset
+        #[weight = <T as Config>::WeightInfo::add_mandatory_mediators(mediators.len() as u32)]
+        pub fn add_mandatory_mediators(
+            origin,
+            ticker: Ticker,
+            mediators: BoundedBTreeSet<IdentityId, T::MaxAssetMediators>
+        ) {
+            Self::base_add_mandatory_mediators(origin, ticker, mediators)?;
+        }
+
+        /// Removes all identities in the `mediators` set from the mandatory mediators list for the given `ticker`.
+        ///
+        /// # Arguments
+        /// * `origin`: The secondary key of the sender.
+        /// * `ticker`: The [`Ticker`] of the asset that will have mediators removed.
+        /// * `mediators`: A set of [`IdentityId`] of all the mediators that will be removed from the mandatory mediators list.
+        ///
+        /// # Permissions
+        /// * Asset
+        #[weight = <T as Config>::WeightInfo::remove_mediators(mediators.len() as u32)]
+        pub fn remove_mediators(
```

Comment:
```suggestion
        pub fn remove_mandatory_mediators(
```
To keep the same naming as the add extrinsic.